src/instruments/src/Common/simVars.tsx

```diff
@@ -31,11 +32,26 @@ const normalizeUnitName = (unit: UnitName): UnitName => {
         case "Knots":
         case "knots":
             return "knots";
+        case "String":
+        case "string":
+            return "string";
         default:
             return unit;
     }
 }
 
+const generateMockValue = (unit: UnitName) => {
+    return ({
+        'bool': !!random(0, 1),
```

Comment:
Just use Math.random instead.

Reply:
Math.random works completely differently

Reply:
`Math.random()` returns a random decimal between 0 and 1, so if you want a random number between 0 and 1 just `Math.round(Math.random())`.
If you need a random number between 0 and 5000 do `Math.round(Math.random() * 5000)`. 
If you need a random number between 100 and 250 do `100 + Math.round(Math.random() * 150)`.

Reply:
Yep I know , but what's the point of repeating that instead of using simple function . That only decreases readability

Reply:
Actually I think this is less readable than a Math.random equivalent because it's not clear that random(0, 1) returns an int rather than a float unless you're familiar with lodash's API. On first inspection it looks like you've created a generator that almost never returns false.